planetExpress/Game.java

```diff
@@ -0,0 +1,50 @@
+package planetExpress;
+import java.util.Scanner;
+public class Game {
+
+	public static void main(String[] args) 
+	{
+		PlanetExpress planet;
+		Score score;
+		Boolean stable = true;
+		
+		 planet = new PlanetExpress();
+		 score = new Score();
+		 Scanner reader = new Scanner(System.in);
+		 
+		 
+		 while (stable = true)
+		 {
+			System.out.println("Choose what do you want to do next:");
+			System.out.println("press \"1\" if you want to deliver");
+			System.out.println("press \"2\" if you want to steal");
+			System.out.println("press \"3\" if you want to eat");
+			System.out.println("press \"4\" if you want to drink");
+			System.out.println("press \"5\" if you want to account");
+			int step = reader.nextInt();
+			
+			switch(step)
+			{
+				case '1' :
```

Comment:
Since the value is an integer, I don't think you need `'1'`, I think you can just use plain old `1`